externs/browser/w3c_audio.js

```diff
@@ -992,18 +1437,16 @@ function webkitAudioPannerNode() {}
  * @constructor
  * @extends {HTMLAudioElement}
  */
-function Audio(src) {}
+function Audio(src) {};
```

Comment:
Here and elsewhere function declarations shouldn't have terminating `;`. You only need those when you're assigning a function to a variable or property.